### DISCUSSION TITLE
Is it possible to switch flags when committing?

### DISCUSSION THREAD

**Discussion by isgj** [jesseduffield/lazygit#2241] [Q&A] [answered]
Hello, is it possible to switch flags when committing (amending)?

Something like this
![image](https://user-images.githubusercontent.com/20517048/199203292-a8ebaceb-57e1-49ca-8340-8782e3d2955a.png)

**Comment by mark2185** [comment, marked as answer]
Currently only through a `custom command`. Either a predefined one, or `:git commit <flags>`.

**Reply by isgj** [reply]
ok, thanks
